lib/index.js

```diff
@@ -34,61 +29,21 @@ module.exports = class SpikeDatoCMS {
     })
 
     compiler.plugin('emit', (compilation, done) => {
-      if (this.json) {
-        writeJson(compilation, this.json, this.addDataTo.dato)
-      }
-
-      // if cache is set and cache file doesn't exist; write it
-      if (this.cache && this.rewrite) {
-        writeCache(this.cache, this.addDataTo.dato)
```

Comment:
Well, DatoCMS pushes some websocket notifications everytime the content changes:

```js
const SiteChangeWatcher = require('datocms-client/dump/SiteChangeWatcher');
const watcher = new SiteChangeWatcher(site.id);
watcher.connect(function() { 
  console.log('Data changed!'); 
});
```

So I think we should definitely get rid of the `cache` option, and automatically cache between these notifications.